sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -243,11 +252,13 @@ export class Recorder {
    *
    * Helps in redirecting the requests to the proxy tool instead of directly going to the service.
    */
-  public configureClientOptionsCoreV1<
-    T extends {
+  public configureClientOptionsCoreV1<T>(
```

Comment:
Curious as to why this change was needed? IMO the way it was before is a tad more readable (not that I have that strong an opinion).

Reply:
that was interpreted differently than this.

Reply:
Yeah, just wondering how exactly it's different.

Reply:
Example

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgPIAczAPYgM7IDeycAXMiAK4C2ARtADTK3l5hSgDmyAvgFB8YlEAiy5kCXDGCdKUCAGEANsAjgMY-AB4AKsggAPSCAAmBYtkw58AfnIbreANoBdXgD4AFJc15yOgEp-Ij5kMOR5MDkQZB9HAG4+fj5JEGlZeWVVdStcPE8LXPxyJ2IyZABmJhZkACI4WgRa3hcmBHJ6xpNmngCBQWFRawkpGTlFFTUwBzyAJl0vOLzggDIiWKK8OzRN114g5D01wt9tmfw9nhDwiIgoqBil-ETkgZFNCXk4SCypzwRJjlfORiO1kGwOCBOEwnlt7Ls3L1ruEAdlpptEjdItEXv0Ul8foCwP9RhkJmjzvlQR0GghuvsAokBAgCeS-ql0uNfkDHLMChIaV0egFGXwgA